packages/editor/yarn.lock

```diff
@@ -541,49 +541,37 @@ tslib@^2.1.0:
   resolved "https://registry.yarnpkg.com/tslib/-/tslib-2.1.0.tgz#da60860f1c2ecaa5703ab7d39bc05b6bf988b97a"
   integrity sha512-hcVC3wYEziELGGmEEXue7D75zbwIIVUMWAVbHItGPx0ziyXxrOMQx4rQEVEV45Ut/1IotuEvwqPopzIOkDMf0A==
 
-tslib@^2.2.0:
-  version "2.2.0"
-  resolved "https://registry.yarnpkg.com/tslib/-/tslib-2.2.0.tgz#fb2c475977e35e241311ede2693cee1ec6698f5c"
-  integrity sha512-gS9GVHRU+RGn5KQM2rllAlR3dU6m7AcpJKdtH8gFvQiC4Otgk98XnmMU+nZenHt/+VhnBPWwgrJsyrdcw6i23w==
-
-uniforms-bridge-json-schema@3.0.x:
-  version "3.0.1"
-  resolved "https://registry.yarnpkg.com/uniforms-bridge-json-schema/-/uniforms-bridge-json-schema-3.0.1.tgz#5ae5c3c97afb2bcac78f1467068c3862ede05822"
-  integrity sha512-ofdQdMHwV7rB9F4dVCU4iKQogFIWCYLi6G0FUJsFJwv++iN0jCYbRVXA7mRTs++W0OGpw6GCaJ7D9LQHObwPPg==
+"uniforms-bridge-json-schema@^2.6.0 <=3.4.0":
+  version "2.6.9"
+  resolved "https://registry.yarnpkg.com/uniforms-bridge-json-schema/-/uniforms-bridge-json-schema-2.6.9.tgz#504a7a121d310795036a24967426750895d4bae6"
+  integrity sha512-cZL2EBrhzRRsZ4M7gUvpTLaKc+PYCJ4r1awxQyI7j18V0m21Ir0kDh1zoTDVqvi+S2uIHDjoy4ey9ukki4zeYA==
   dependencies:
     invariant "^2.0.0"
     lodash "^4.0.0"
     tslib "^1.10.0"
-    uniforms "^3.0.1"
+    uniforms "^2.6.9"
 
-uniforms-material@3.0.x:
-  version "3.0.1"
-  resolved "https://registry.yarnpkg.com/uniforms-material/-/uniforms-material-3.0.1.tgz#aac5ef3851b0ece1bbb39ca2052e367e64195a54"
-  integrity sha512-7J1Cf7cskc4s0pAbQgpwU3VZliX5fTLki61GAFrFW+rN6Rky5mNtJB28adFQXFAHK5Eg34hqPe+VDKG0OxCCJw==
+"uniforms-material@^2.6.0 <=3.4.0":
+  version "2.6.9"
+  resolved "https://registry.yarnpkg.com/uniforms-material/-/uniforms-material-2.6.9.tgz#525fc77c0132bc11a30d67769b16d808149aae63"
+  integrity sha512-WSLc8L7CjVkbEAtgohLzwtRomyFEn0JbJxqE+7OANt1UoIUWu01kxNcU9KSCcmEz/LdTUZ8zB5GUGGnUzBMsNg==
   dependencies:
     invariant "^2.0.0"
     lodash "^4.0.0"
+    prop-types "^15.5.7"
     tslib "^1.10.0"
-    uniforms "^3.0.1"
+    uniforms "^2.6.9"
 
-uniforms@3.0.x:
-  version "3.0.1"
-  resolved "https://registry.yarnpkg.com/uniforms/-/uniforms-3.0.1.tgz#501516bec1e556d815275e4bf8ea5b18b1f38078"
-  integrity sha512-ULYBRs0GUc1HGKY+TpOIeYQnCFdy25CDUFkPqrlYlNU45YnommuadFtl3o6TyeOytY4z7M6Hy0SgDSeWYYZoLQ==
+"uniforms@^2.6.0 <=3.4.0", uniforms@^2.6.9:
+  version "2.6.9"
+  resolved "https://registry.yarnpkg.com/uniforms/-/uniforms-2.6.9.tgz#0741532cf82331d5e432f0fb5547ac24713965ba"
+  integrity sha512-RnKx276c4xS5yoo8MxH2EtZbmzpsb5vFWP+VYCMC8tH/VSK4jlVBgCbCTADbMlcojwnAOGjUrA+m3s4tacdLbg==
```

Comment:
it seems it downgraded the version to 2.6.9. that is probablyy the problem you are facing. Altough i am not sure if this yarn.lock file is used at all (maybe it is used in the demo project, as it is a lerna mono-repo). 

nevertheless, you can safely revert this yarn.lock change. the change in the package.json is the only required change.

optionally we can add resolutions field to force a version for the demo project. npm installations are not affected by this. they only look for the version in the package.json (and their own local lock files of couse)

Reply:
Yeah, I was thinking the same. But wondering why it did the downgrade with a fresh install in the first place. I will probably read up on this. Because if this is the case, then people who just clone and install won't be able to work on this. Also, that means the package is not in fact compatible with 2.6.9 (or to be more precise the types aren't ) so I am wondering if we should not abandon the 2nd major version support. Would this even be a problem for people? It seems reasonable. 

I don't like the resolutions approach, it often backfires when people forget it's there.